integration-tests/tests/run-workflow.integration-test.js

```diff
@@ -0,0 +1,56 @@
+const pRetry = require('p-retry')
+const { testUrl, workflowName, billingProject } = require('../utils/integration-config')
+const { withWorkspace } = require('../utils/integration-helpers')
+const { click, clickable, findElement, input, signIntoTerra, waitForNoSpinners, findInGrid, workspaceTab } = require('../utils/integration-utils')
+
+
+const testEntity = { name: 'test_entity_1', entityType: 'test_entity', attributes: { input: 'foo' } }
+
+test('run workflow', withWorkspace(async ({ workspaceName }) => {
+  await page.goto(testUrl)
+  await signIntoTerra(page)
+
+  await page.evaluate((name, billingProject, testEntity) => {
+    return window.Ajax().Workspaces.workspace(billingProject, name).createEntity(testEntity)
+  }, workspaceName, billingProject, testEntity)
+
+  await click(page, clickable({ textContains: 'View Workspaces' }))
+  await click(page, clickable({ textContains: workspaceName }))
+
+  await click(page, workspaceTab('workflows'))
+  await waitForNoSpinners(page)
+  await click(page, clickable({ textContains: 'Find a Workflow' }))
+  await click(page, clickable({ textContains: workflowName }))
+  await waitForNoSpinners(page)
+  await click(page, clickable({ textContains: 'Add to Workspace' }))
+  // note that this automatically brings in the highest numbered config, which isn't what happens when going through the method repo in FC
+
+  await waitForNoSpinners(page)
+  await click(page, clickable({ textContains: 'Select Data' }))
+  await click(page, input({ labelContains: 'Choose specific rows to process' }))
+  await click(page, `//*[@role="checkbox" and contains(@aria-label, "${testEntity.name}")]`)
+  await click(page, clickable({ textContains: 'OK' }))
+  await click(page, clickable({ textContains: 'Run analysis' }))
+
+  await Promise.all([
+    page.waitForNavigation(),
+    click(page, clickable({ textContains: 'Launch' }))
+  ])
+
+  await pRetry(async () => {
+    try {
+      await findInGrid(page, 'Succeeded')
+    } catch (e) {
+      throw new Error(e)
+    }
+  }, {
+    onFailedAttempt: async () => {
+      await page.reload()
+      await signIntoTerra(page)
+    }
+  })
+
+  await click(page, workspaceTab('data'))
+  await click(page, clickable({ textContains: 'test_entity' }))
+  await findElement(page, `//*[@role="grid"]//*[contains(.,"${testEntity.name}")]/following-sibling::*[contains(.,"result: ${testEntity.attributes.input}")]`)
```

Comment:
2 questions:
1. This looks like the first instance of a findElement in a test that directly specifies an Xpath. Would it make sense to create a helper for this? 

2. Is this intended to find a second element on the same row, or anywhere in the grid?

Reply:
Same row.

Reply:
Also made a helper

Reply:
TOL: Thanks. While it isn't something we should address now - would this match any item in the table, even on a different row, given the way the markup is currently setup in the DOM?

I just wanted to put this out there as a thought as I am wondering if/how we should address this in the future.